### PR TITLE
Add MinIO example configuration

### PR DESCRIPTION
**Pull request by lkmhaqer**
This example shows a basic configuration, where users are given all admin access to MinIO.

### CODE REVIEW

File: example_configs/minio.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+### Navigation
+
+- Login to the WebUI as a consoleAdmin user
+- Navigate to `Administrator > Identity > LDAP`
+- Click `Edit Configuration`
+
+### Configuration Options
+
+- Server Insecure: `Enabled`
```
Comment (nitnelave) [inline example_configs/minio.md:15]: Does it work with LDAPS? We might want to make a note here.

File: example_configs/minio.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+- Server Address: `Hostname or IP for your LLDAP host`
```
Comment (nitnelave) [inline example_configs/minio.md:16]: Small note: I would put the values between backticks when it's a literal value (such as `Enabled` or `uid=admin`), but not when it's a description of the value

File: example_configs/minio.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+- Lookup Bind DN: `uid=admin,ou=people,dc=example,dc=com`
+  - It is recommended that you create a separate user account (e.g, `bind_user`) instead of `admin` for sharing Bind credentials with other services. The `bind_user` should be a member of the `lldap_strict_readonly` group to limit access to your LDAP configuration in LLDAP.
+- Lookup Bind Password: `The password for the user referenced above`
+- User DN Search Base: `ou=people,dc=example,dc=com`
+- User DN Search Filter: `(&(uid=%s)(memberOf=cn=minio_admin,ou=groups,dc=example,dc=com))`
```
Comment (nitnelave) [inline example_configs/minio.md:21]: minio_user, not minio_admin

File: example_configs/minio.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  - This search filter will only allow users that are members of the `minio_user` group to authenticate. To allow all lldap users, this filter can be used instead `(&(uid=%s)(objectClass=person))`
```
Comment (nitnelave) [inline example_configs/minio.md:22]: The alternate filter can be simplified to `(uid=%s)`

**Review by nitnelave — APPROVED** [review]
Looks great :) Don't forget to also add a link to the README!

File: example_configs/minio.md (hunk 1 above)
Comment (lkmhaqer) [reply example_configs/minio.md:15]: Ah, I have not tested this yet.

File: example_configs/minio.md (hunk 3 above)
Comment (lkmhaqer) [reply example_configs/minio.md]: Updated the PR with this feedback!

File: example_configs/minio.md (hunk 4 above)
Comment (lkmhaqer) [reply example_configs/minio.md]: Updated the PR with this feedback!

**Comment by lkmhaqer** [comment]
Trying to figure out how the Table of Contents updated with my README change, is this okay as-is/amended?

File: example_configs/minio.md (hunk 2 above)
Comment (nitnelave) [reply example_configs/minio.md:16]: I think you missed the "Hostname or IP" and the password description.

File: example_configs/minio.md [hunk 5]
```diff
@@ -34,4 +34,4 @@ Now, you can enable LDAP authentication by clicking the `Enable LDAP` button, a
 Creating MinIO policies is outside of the scope for this document, but it is well documented by MinIO [here](https://min.io/docs/minio/linux/administration/identity-access-management/policy-based-access-control.html). Policies are written in JSON, are extremely flexible, and can be configured to be very granular. In this example we will be using one of the built-in Policies, `consoleAdmin`. We will be applying these policies with the `mc` command line utility.
 
 - Alias your MinIO instance: `mc alias set myMinIO http://<your-minio-address>:<your-minio-api-port> admin <your-admin-password>`
-- Attach a policy to your LDAP group: `mc admin policy attach myMinIO consoleAdmin --group='cn=minio_admin,ou=groups,dc=example,dc=com'`
+- Attach a policy to your LDAP group: `mc admin policy attach myMinIO consoleAdmin --group='cn=minio_user,ou=groups,dc=example,dc=com'`
```
Comment (nitnelave) [inline example_configs/minio.md:37]: I believe this one should stay as `minio_admin`, since you're giving it admin rights.

**Comment by nitnelave** [comment]
Thank you everyone for the contribution!
This feature is now available in the latest release, [v0.5.0](https://github.com/lldap/lldap/releases/tag/v0.5.0).
You can support LLDAP by starring our repo, contributing some configuration examples and becoming a sponsor.

**Pull request merged**